series-tvdb.en.yaml

```diff
@@ -2115,3 +2115,12 @@ entries:
         start: 7
       - season: 3
         anilist-id: 122292
+        
+  - title: "Rurouni Kenshin"
```

Comment:
What's the name shown on Plex in TVDB episode order? Does it have the year in it or not?

I don't want to cause conflicts for people who have the old series in their library.

Reply:
Rurouni Kenshin (2023)
[https://thetvdb.com/series/rurouni-kenshin-2023](url)

Reply:
Then the title should be `Rurouni Kenshin (2023)` and the title without the year should not be part of this entry.

Reply:
Both entries should exist no? but with the exact name of TVDB
Especially considering how popular the franchise is

![image](https://github.com/RickDB/PlexAniSync-Custom-Mappings/assets/10836780/14624019-1086-4338-8e4e-d49b2e7051db)

![image](https://github.com/RickDB/PlexAniSync-Custom-Mappings/assets/10836780/7d19e29f-4e18-45a1-927f-9a9f8c320f1d)

Reply:
So is there a way to get them to work? I mean it is as simple as changing the default plex name to add (2023) to the end of the title, but is there a way to get them both to work natively without changing plex titles?

Reply:
I'm waiting for you to do the following:
1. Remove "Rurouni Kenshin" from the 2023 entry
2. Create a separate entry for the old series

If you use TVDB for episode naming, they should show up with different names in Plex.

However, if you use TMDB for episode naming, they're both called the same in Plex so you can only have one of both in your mapping file.

Since you're only editing the TVDB mapping, both entries can be used at the same time.

Reply:
Oh nevermind, I just tested it: Plex turns it into "Rurouni Kenshin".

So as you said, you can only have one of both series in your library without renaming.

Reply:
Okay, so unfortunately this wont work as added so I assume best for me to close the pull request?

Reply:
I've implemented the `guid` feature so this can be handled properly now.